website/manual.md

```diff
@@ -230,7 +223,7 @@ git submodule update
 # to the system provided ones (for packagers of deno ...).
 ./tools/setup.py --no-binary-download
 export DENO_BUILD_ARGS="clang_base_path=/usr clang_use_chrome_plugins=false"
-DENO_GN_PATH=/usr/bin/gn DENO_NINJA_PATH=/usr/bin/ninja ./tools/build.py
+DENO_GN_PATH=/usr/bin/gn DENO_NINJA_PATH=/usr/bin/ninja cargo build
```

Comment:
Shouldn't this be

```suggestion
DENO_NO_BINARY_DOWNLOAD=1 GN=/usr/bin/gn NINJA=/usr/bin/ninja cargo build
```

without the `./tools/setup.py --no-binary-download` line?

However this functionality for `cargo` depends on #2836.

Reply:
I think I want to land this before merging in cargo_gn (#2836), so these should stay the same.

@chrmoritz It would be helpful if you could try this branch locally and make sure it works for you.

Reply:
I can confirm that it still works for me locally, but we now need a way to tell `cargo` to pass `--no-binary-download` to `setup.py`. And explicitly calling `setup.py` here is still unnecessary.

Reply:
Thanks for testing

Reply:
Thanks for adding `DENO_NO_BINARY_DOWNLOAD`!

So now only the docs need some changes for it:
```
# Skip downloading binary build tools and point the build
# to the system provided ones (for packagers of deno ...).
export DENO_BUILD_ARGS="clang_base_path=/usr clang_use_chrome_plugins=false"
DENO_NO_BINARY_DOWNLOAD=1 DENO_GN_PATH=/usr/bin/gn DENO_NINJA_PATH=/usr/bin/ninja cargo build
```

Reply:
done